### DISCUSSION TITLE
Links sesión del 17/02

### DISCUSSION THREAD

**Discussion by AxelFBGarcia** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#15] [Q&A] [answered]
¿Alguien sabe dónde puedo encontrar los links que dieron en la sesión para diseño?

**Comment by YomairaBarrera** [comment, marked as answer]
El MC todavía no ha publicado pero si checas en el teams hay un compañero que ya los publico en un comentario en el post del live.

**Reply by AxelFBGarcia** [reply]
Muchas gracias!!

**Reply by nsaenz** [reply]
Te dejo algunos de los que guarde:

https://www.nngroup.com/articles/ten-usability-heuristics/

https://developer.apple.com/design/human-interface-guidelines/

https://material.io/design

https://coolors.co/222222-ffffff-4b4e6d-84dcc6-95a3b3

https://contrastchecker.com/

Buen fin!
